packages/@react-spectrum/form/stories/Form.stories.tsx

```diff
@@ -147,7 +166,7 @@ export const FieldsWithAutoCompleteProperty = () => {
             label="City"
             marginEnd="size-100"
             flex={1} />
-          <Picker
+          <Picker<State>
```

Comment:
That's not good. Is the inference broken?

Reply:
I'm not sure what happened here, might've changed something that fixed this later in the process, i was able to revert these